internal/driver/common/canvas.go

```diff
@@ -191,6 +191,15 @@ func (c *Canvas) Focused() fyne.Focusable {
 	return mgr.Focused()
 }
 
+// Preeditable return the current preeditable object only when focused
```

Comment:
This will crash if the input is not Preeditable. It should not be needed as it will always be the same object returned by `Focused()`

Reply:
@andydotxyz It may be counter-intuitive to you, but this code works fine. Unlike C casts, Go interface casts return nil if the interface is not satisfied.

Reply:
It would only happen if you started to interact with an input that had not implemented that interface.
Something like the checkbox I would think for example.